modules/http/server/src/test/scala/Utils.scala

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc.http
```

Comment:
Just a suggestion for incoming comments below:

```diff
-package freestyle.rpc.http
+package freestyle.rpc
+package http
```

Reply:
Sure, I can make the change. I guess the convention is the project's root packages on one line and the module subpackages on the next?

Reply:
Well, after reading up on this, I discover that the multiline style brings everything on each line in scope. Somehow I never knew about this system event though it's 8 years old by now: https://www.artima.com/scalazine/articles/chained_package_clauses_in_scala.html. I guess we can learn something new everyday. :-)

Annoyingly since we have a `freestyle.rpc.fs2` packages it means we need to prefix all our `fs2` imports with `_root_`, doesn't look too pretty. I can still set it up like that, for convention's sake.